tests/Baseline/hilti.rt.profiler/output

```diff
@@ -2,4 +2,4 @@
 #name count avg-% total-%
 hilti/func/Foo::fibo 192
 hilti/func/Foo::y 95
-hilti/total 1 100.00 100.00
+hilti/total 1 1.00e+02 1.00e+02
```

Comment:
This feels much harder to read. What about instead increasing the precision for the fractional part?

Reply:
I agree that this reads less nicely. I think no matter how much we increase the number of digits we show we will still loose data at some point with that approach for small fractions. With the current output format I see ~1/3 of entries with `tot` serialized as `0.00`. If we want to avoid exponents (which requires slightly more work to compare magnitudes), what do you think about e.g., just showing all double digits?

Overall my gut feeling is that since such output has so much data in it, users would always want to write quick ad-hoc tooling to look at it (I used pandas after converting to csv).

Reply:
Do those entries really matter? If yes, then how about adding a new mode that provides an alternative output format for parsing, like CSV with full precision?

Reply:
Good question. The perf output I am looking at ATM has >1500 entries, so I hadn't really consider this anything but a "full format" which can only be consumed with additional tooling 😅

Reply:
For me, that tooling has so far been `grep/awk/sort`, and in the end I still look at it. :-)

Reply:
Fair enough, and if that is the consensus I am totally for keeping the current format. I initially wanted this to fix `avg` which mostly showed `0.00`, but that is easy to compute with better precision offline.